### PR TITLE
17 create drawer menu

### PR DESCRIPTION
**Pull request by Pholenk**
this PR refers to #17 
feel free to review the codes

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { store, persistor } from './src/store';
 import UserRegister from './src/containers/UserRegister';
 // import SendPackagePage from './src/components/sendPackagePage';
 import UserLogin from './src/containers/UserLogin';
-import Onboarding from './src/components/onboarding';
+import Onboarding from './src/components/Onboarding';
```
Comment (Pholenk) [inline App.js]: Ini ditempat gw error kalo pake uppercase

Comment (mazsam) [reply App.js]: iya mas @Pholenk sama

Comment (kru) [reply App.js]: `git mv src/components/onboarding src/components/tmp` -> `git mv src/component/tmp -> Onboarding`

tinggal ganti cara panggilnya.

**Comment by Pholenk** [comment]
@kru ini udah gua bikin full-width

**Comment by kru** [comment]
thank you @Pholenk!

**Comment by Pholenk** [comment]
Don't forget to close the issue @kru

**Pull request merged**